core/esmf-aspect-model-resolver/src/test/resources/samm_1_0_0/org.eclipse.esmf.test/2.0.0/BammTest.ttl

```diff
@@ -23,4 +23,5 @@
    bamm:operations ( ) .
 
 :property a bamm:Property ;
+   bamm:name "TestProperty" ;
```

Comment:
The value of `bamm:name` must be identical to the URN's local part, i.e. in this case "property".